deployment/v3/utils/prop_migrator/knowledge/property-categories.csv

```diff
@@ -1,4 +1,760 @@
 Property file name,Key,Property Category
+admin-default.properties,authmanager.base.url,URL
+admin-default.properties,mosip.admin.accountmgmt.auth-manager-base-uri,URL
+admin-default.properties,mosip.kernel.signature.cryptomanager-encrypt-url,URL
+admin-default.properties,auth.server.validate.url,URL
+admin-default.properties,auth.server.refreshToken.url,URL
+admin-default.properties,javax.persistence.jdbc.url,URL
+admin-default.properties,javax.persistence.jdbc.password,Key
+admin-default.properties,auth.server.user-register-url,URL
+admin-default.properties,mosip.kernel.emailnotifier-url,URL
+admin-default.properties,auth.server.sendotp-url,URL
+admin-default.properties,auth.server.user-add-password-url,URL
+admin-default.properties,mosip.admin.security.policy.userrole-auth-url,URL
+admin-default.properties,mosip.admin.uinmgmt.uin-detail-search,URL
+admin-default.properties,mosip.mosip.kernel.packet-status-update-url,URL
+admin-default.properties,mosip.kernel.packet-reciever-api-url,URL
+admin-default.properties,mosip.kernel.zone-validation-url,URL
+admin-default.properties,mosip.kernel.audit.manager.api,URL
+admin-default.properties,mosip.kernel.masterdata.audit-url,URL
+admin-default.properties,mosip.admin-services.audit.manager.api,URL
+admin-default.properties,mosip.open-id.base-url,URL
+admin-default.properties,mosip.admin-services.open-id,clientsecret,Key
+admin-default.properties,mosip.admin-services.redirecturi,URL
+admin-default.properties,mosip.admin-services.open-id.authorization_endpoint,URL
+admin-default.properties,mosip.admin-services.open-id.token_endpoint,URL
```

Comment:
I see the below as duplicate but from different files, can you check this once ?

syncdata.websub.callback.secret.ca-cert

Also generally ensure there are no duplicate in these files

Reply:
The above property file is present in "kernel-default.properties" and "syncdata-default.properties" sir, i have cross verified.